BeeKit/HeathKit/Toothbrushing/ToothbrushingDailyMinutesHealthKitMetric.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
+import HealthKit
+
+/// tracks toothbrushing, in number of (decimal) minutes per day (daystamp)
+class ToothbrushingDailyMinutesHealthKitMetric: CategoryHealthKitMetric {
+    private static let healthkitMetric = ["toothbrushing", "minutes-per-day"].joined(separator: "|")
+    
+    private init(humanText: String,
+                 databaseString: String,
+                 category: HealthKitCategory) {
+        super.init(humanText: humanText,
+                   databaseString: databaseString,
+                   category: category,
+                   hkSampleType: HKObjectType.categoryType(forIdentifier: .toothbrushingEvent)!)
+    }
+    
+    override func units(healthStore : HKHealthStore) async throws -> HKUnit {
+        HKUnit.second()
+    }
+    
+    static func make() -> ToothbrushingDailyMinutesHealthKitMetric {
+        .init(humanText: "Teethbrushing (in seconds per day)",
+              databaseString: healthkitMetric,
+              category: HealthKitCategory.SelfCare)
+    }
```

Comment:
Let's remove this. I'd like all these constants to remain in `HealthKitConfig.swift` for consistency.